harborclient/harborclient.py

```diff
@@ -1,13 +1,16 @@
 #!/usr/bin/env python
 
-import requests
-import simplejson
+# for inner
+import json
 import logging
+# for thirdparty
```

Comment:
if include simplejson some unicode parse error will happen，and the protocol of json is the same with simplejson. The best way is replace simplejson to json